mmedit/models/mattors/dim.py

```diff
@@ -67,7 +67,6 @@ def _forward(self, x, refine):
             pred_refine = self.refiner(refine_input, raw_alpha)
         else:
             pred_refine = None
-
```

Comment:
Same as above~

Reply:
> Same as above~

Sorry, it's my fault, I have retrieved this change.